.github/workflows/ci.yaml

```diff
@@ -80,6 +80,7 @@ jobs:
           # Setting RUSTDOCFLAGS because `cargo doc --check` isn't yet implemented (https://github.com/rust-lang/cargo/issues/10025).
           RUSTDOCFLAGS: "-D warnings"
       - uses: actions/upload-artifact@v3
+        if: ${{ matrix.os == 'ubuntu-latest' }}
```

Comment:
We only need the linux binary. Trying to run this on windows creates a warning because the file is named `ruff.exe` and not `ruff`